dev/source-release.sh

```diff
@@ -36,7 +36,8 @@ usage () {
 }
 
 # Default repository remote name
-remote="apache"
+# Use origin so this can run on ASF infra or anywhere that directly clones the repository.
+remote="origin"
```

Comment:
Alternatively, we can add upstream as apache or ensure that it's installed that way.